app/views/security_group/edit.html.haml

```diff
@@ -1,6 +1,12 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "securityGroupFormController"}
+%form#form_div{:name           => "angularForm",
+   "ng-controller" => "securityGroupFormController as vm",
```

Comment:
As I commented earlier, this is a style matter and I tend to try to be consistent.